bootloader/src/pyi_utils.c

```diff
@@ -925,6 +929,10 @@ pyi_utils_create_child(const char *thisfile, const ARCHIVE_STATUS* status,
 
     argv_pyi = (char**)calloc(argc + 1, sizeof(char*));
     argc_pyi = 0;
+    if (!argv_pyi) {
+        VS("LOADER: failed to allocate argv_pyi: %s\n", strerror(errno));
```

Comment:
I think it'd be better if error messages were printed using `FATALERROR()` or `OTHERERROR()` instead of `VS()`, so that they are displayed even if non-debug bootloader is used.

Reply:
I agree. I did not know about the other logging facilities -- I will do so -- it is useful for sure to be informed of all errors in the non-debug case for sure.  Will update.

Reply:
Ok, I pushed a commit to do `FATALERROR` and/or `OTHERERROR` in some cases.

Reply:
@rokm `VS` calls are displayed under any bootloader, however `FATALERROR` calls pop up with a dialog on windowed binaries, whereas `VS` calls do not. I admit to not having encountered `OTHERERROR` before, but I'm not overly surprised - I've yet to start to tackle some of the larger bootloader bugs.

Reply:
@Legorooj  Hmm. VS calls appear to be compiled-out to nothing on all non-debug bootloaders:

<img width="529" alt="Screen Shot 2020-10-28 at 12 46 37 PM" src="https://user-images.githubusercontent.com/266627/97426131-b1a31b00-191b-11eb-90a0-4b1efd5783ab.png">

FATALERROR does indeed pop up a window on ms-windows only, and only in the WINDOWED case.  On Linux or on OSX, there they just printf to console in all cases.

Reply:
Oh yeah sorry, ignore me. I haven't touched the bootloader much yet, and I get regularly confused with it 😅. I need to sharpen my C skills and fix a lot of the problems the bootloader has. (Ideally I'd like to reprogram it in Rust, but that requires me to fully understand everything going on in the bootloader first).

Reply:
Ah ok, no worries! 👍